src/lib/server-utilities.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import crypto from 'crypto';
+
+export function md5Hash(file) {
+  const hashInstance = crypto.createHash('md5');
+  // readFileSync in the syncronous version of readFile
+  const fileContents = fs.readFileSync(file, 'utf8');
+  return hashInstance.update(fileContents).digest('hex');
+}
```

Comment:
I think what I've become used to from other projects is using (is that called Pascal case?) the one where you camelCase but with the first letter capitalized as well, and for folder-names and normal js files use `-`. But yeah I completely agree on consistency.

What do you say to renaming the other files to `-`. Or should I just change it to camelCase?

Also what I've gotten used to and seen Dan Abramov's opinion on it is not using `.jsx` (the argument is with the current JS environment should you then rename the extension of files for every library / framework you use on every file? `.flux` `.js(whatever)` etc.)
Which I guess we could also quickly change if we wanted but I'm also find with our current naming convention and just keeping it.